lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -344,6 +344,19 @@ public static function duplicateDiscriminatorEntry(string $className, array $ent
         );
     }
 
+    ///**
+    // * @param class-string $rootEntityClass
+    // * @param class-string $childEntityClass
+    // */
+    //public static function missingInheritanceTypeDeclaration(string $rootEntityClass, string $childEntityClass): self
+    //{
+    //    return new self(sprintf(
+    //        "Entity class '%s' is a subclass of the root entity class '%s', but no inheritance mapping type was declared.",
+    //        $childEntityClass,
+    //        $rootEntityClass
+    //    ));
+    //}
```

Comment:
Let's do the switch to the new code suggested by @mpdude since it's causing issues?